src/Microsoft.DotNet.ProjectModel/WorkspaceContext.cs

```diff
@@ -280,6 +351,20 @@ private FileModelEntry<LockFile> AddLockFileEntry(string projectDirectory, FileM
             return currentEntry;
         }
 
+        private ProjectContextBuilder InitializeProjectContextBuilder(Project project)
```

Comment:
In both cases where this function is used, the direct following action is to build the `ProjectContext`. So how about return the `ProjectContext` collection here directly?